content/dstu2/allergy-intolerance.md

```diff
@@ -0,0 +1,128 @@
+---
+title: AllergyIntolerance | FHIR DSTU 2 API
+---
+
+# AllergyIntolerance
+
+* TOC
+{:toc}
+
+## Search
+
+Search for AllergyIntolerances that meet supplied query parameters:
+
+    GET /AllergyIntolerance?:parameters
+
+### Parameters
+
+ Name    | Required? | Type                                                           | Description
```

Comment:
I think we need a new table, Terminology Bindings. Similar to: http://hl7.org/fhir/DSTU2/allergyintolerance.html#tx

But we would document the bindings for the non-required valuesets. This will eventually need to be throughout the resources, but it's definitely needed on the resources allowing write/update. Note that the binding doesn't change by action, so it can be towards the top/on it's own.